main.go

```diff
@@ -70,6 +72,8 @@ func main() {
 		err = list.List(args)
 	case "login":
 		err = login.NewLogin(args)
+	case "sda-download":
+		err = sdaDownload.SdaDownload(args)
```

Comment:
This shouldn't really  be a new command, we should just have `download` since the current download from exported s3 buckets will be deprecated.

Reply:
Correct but as far as I remember we will keep both for some time and when the other one will be deprecated then for sure we can change the name